modules/templates/static.go

```diff
@@ -47,6 +49,29 @@ func (templates templateFileSystem) Get(name string) (io.Reader, error) {
 	return nil, fmt.Errorf("file '%s' not found", name)
 }
 
+func GetAsset(name string) ([]byte, error) {
```

Comment:
lint err

Reply:
I have changed this one. But it seems lint check on this file is ignored because of some reason.